supporter-product-data/src/main/scala/com/gu/lambdas/QueryZuoraLambda.scala

```diff
@@ -19,17 +20,18 @@ class QueryZuoraLambda extends Handler[QueryZuoraState, FetchResultsState] {
 
 }
 
-object QueryZuoraLambda {
+object QueryZuoraLambda extends StrictLogging {
   def queryZuora(stage: Stage, queryType: QueryType) = {
-    SafeLogger.info(s"Attempting to submit ${queryType.value} query to Zuora")
+    logger.info(s"Attempting to submit ${queryType.value} query to Zuora")
+    val attemptedQueryTime = LocalDateTime.now.minusMinutes(1)
     for {
       config <- ZuoraQuerierConfig.load(stage)
       service = new ZuoraQuerierService(config, configurableFutureRunner(60.seconds))
       result <- service.postQuery(queryType)
```

Comment:
presumaably todo pass the attemptedQueryTime into the query

Reply:
ok I hve got it now - we pass in the previous time from the config object already - the current query attemptedQueryTime we just have to guess conservatively as they don't tell us the current session time.